src/nni_manager/training_service/reusable/channels/amlCommandChannel.ts

```diff
@@ -40,8 +39,9 @@ export class AMLCommandChannel extends CommandChannel {
 
     public async start(): Promise<void> {
         // start command loops
-        this.receiveLoop();
-        this.sendLoop();
+        await Promise.all([
```

Comment:
don't await here. it may cause the whole trial dispatcher waiting. The async call is to wait some must operation during start a channel, like create web socket service. It's not used to wait receive and send loop like this.

Reply:
sure, reverted.

Reply:
since it can not be awaited, should remove the async.

Reply:
the message loops don't need await, but other operations like start service may need.